packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -1498,7 +1498,7 @@ class IrisGrid extends Component<IrisGridProps, IrisGridState> {
     return rowIndex != null ? modelRows.get(rowIndex) : null;
   }
 
-  getTheme(): Partial<IrisGridThemeType> {
+  getTheme(): IrisGridThemeType {
```

Comment:
I see the satisfies on our theme, but does this still work if a third-party provided theme is a partial?

Like we want third parties to be able to declare themes that override just one parameter for example.

Reply:
Yes. This return type was just incorrect. It returns `getCachedTheme` which spreads the default theme and returns a full theme. The return type needed to be fixed for the cell overflow context menu handler